install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,253 @@
+[[contributing-to-docs-templates-topic-template]]
+= Aggregate logging sizing guidelines 
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The https://github.com/openshift/openshift-docs/blob/master/install_config/aggregate_logging.adoc#overview[ElasticSearch, Fluentd and Kibana] (EFK) stack aggregates logs from nodes and applications running inside your openshift installation.
+Once deployed it uses http://www.fluentd.org/architecture[Fluentd] to aggregate event logs from all nodes, projects and pods into https://www.elastic.co/products/elasticsearch[Elasticsearch]. It also provides a centralized https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] web UI where users and administrators can create rich visualizations and dashboards with the aggregated data.
+
+Fluentd https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html[bulk uploads] logs to an index, in json format, and ElasticSearch then routes your search requests to the appropriate shards.
+
+== Installation
+The general procedure for installing an aggregate logging stack in {product-title} is described in https://docs.openshift.com/enterprise/3.2/install_config/aggregate_logging.html[Aggregating Container Logs].
+There are some important things to keep in mind while going through the installation guide:
+
+In order for the logging pods to spread evenly across your cluster, an empty https://docs.openshift.com/enterprise/3.2/admin_guide/managing_projects.html#using-node-selectors[node selector] should be used.
+
+----
+$ oadm new-project logging --node-selector=""
```

Comment:
from a usability perspective using daemon-sets, this is weird to have to provide the node-selector at all. 

/cc @richm